iml-gui/crate/src/lib.rs

```diff
@@ -863,7 +865,7 @@ fn view(model: &Model) -> Vec<Node<Msg>> {
         .els(),
         Page::Filesystem(page) => main_panels(
             model,
-            page::filesystem::view(&model.records, page, &model.locks, model.auth.get_session())
+            page::filesystem::view(&model.records, page, &model.locks, model.auth.get_session(), model.conf.use_stratagem)
```

Comment:
Maybe we should start passing the whole `model`.

Reply:
The thought did cross my mind :) I think 4 parameters are probably ok. If it ends up going beyond that then I think it makes sense to just pass the model. For now I think this is ok though.

Reply:
In general I think it's better to pass in params then expose the parent model to a child component.